From 5f64b702482d5818b8573d12ef1b58af00bbd81d Mon Sep 17 00:00:00 2001
From: Johannes Berg <johannes.berg@intel.com>
Date: Wed, 31 Aug 2022 21:13:11 +0200
Subject: [PATCH] link: fix some formatting

The bss flags has a spurious newline, and we don't use a
tab for indentation after the colon in other places, fix
that here.

Signed-off-by: Johannes Berg <johannes.berg@intel.com>
---
 link.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/link.c b/link.c
index 2074488..31de8b4 100644
--- a/link.c
+++ b/link.c
@@ -181,7 +181,7 @@ static int print_link_sta(struct nl_msg *msg, void *arg)
 			fprintf(stderr, "failed to parse nested bss parameters!\n");
 		} else {
 			char *delim = "";
-			printf("\n\tbss flags:\t");
+			printf("\tbss flags: ");
 			if (binfo[NL80211_STA_BSS_PARAM_CTS_PROT]) {
 				printf("CTS-protection");
 				delim = " ";
@@ -192,9 +192,9 @@ static int print_link_sta(struct nl_msg *msg, void *arg)
 			}
 			if (binfo[NL80211_STA_BSS_PARAM_SHORT_SLOT_TIME])
 				printf("%sshort-slot-time", delim);
-			printf("\n\tdtim period:\t%d",
+			printf("\n\tdtim period: %d",
 			       nla_get_u8(binfo[NL80211_STA_BSS_PARAM_DTIM_PERIOD]));
-			printf("\n\tbeacon int:\t%d",
+			printf("\n\tbeacon int: %d",
 			       nla_get_u16(binfo[NL80211_STA_BSS_PARAM_BEACON_INTERVAL]));
 			printf("\n");
 		}
-- 
2.41.0.694.ge786442a9b-goog

